packages/tasks/src/model-libraries.ts

```diff
@@ -516,6 +516,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	reverb: {
+		prettyLabel: "Reverb",
+		repoName: "Reverb",
+		repoUrl: "https://github.com/revdotcom/reverb",
+		filter: false,
+		countDownloads: `path_extension:"yaml" OR path_extension:"pt" OR path_extension:"bin"`,
```

Comment:
Looks like counting only `config.yaml` would be enough, isn't it? If we consider that loading the model requires to download the config. This way no need to have both .pt and .bin depending on the file format. Also, counting both yaml and weights files will lead to duplicate counts.

Also, good news, `config.yaml` is already counted by default when no rules are set (see [here](https://huggingface.co/docs/hub/en/models-download-stats#which-are-the-query-files-for-different-libraries)). That's why there are already downloads in https://huggingface.co/models?other=reverb

Reply:
```suggestion
```

TL;DR: this line can be removed

Reply:
You're indeed correct. I hallucinated there being no `config.yaml` in speaker diarization repos.